src/Aspire.Hosting.Azure.Storage/AzureStorageExtensions.cs

```diff
@@ -49,6 +49,11 @@ public static IResourceBuilder<AzureStorageResource> AddAzureStorage(this IDistr
                 sku: StorageSkuName.StandardGrs
                 );
 
+            // Unfortunately Azure Storage does not list ACA as one of the resource types in which
```

Comment:
> Unfortunately 

Is there some sort of issue tracking this? Do we need to raise this problem somewhere?